app/_src/mesh/plan-and-deploy/license.md

```diff
@@ -124,6 +124,24 @@ In Universal mode, configure a valid license by using the following environment
 * `KMESH_LICENSE_PATH` - value with the path to a valid license file.
 * `KMESH_LICENSE_INLINE` - value with the actual contents of the license file.
 
+## Update a license
+
+### Kubernetes
+
+1. Update the license 
+  Update the `kong-mesh-license` Secret in `kong-mesh-system` namespace with a new license.
```

Comment:
1. Update the `kong-mesh-license` Secret in the `kong-mesh-system` namespace with a new license:
```
kuma:
    controlPlane:
      secrets:
        - Env: "KMESH_LICENSE_INLINE"
          Secret: "kong-mesh-license"
          Key: "new-license.json"
```

Reply:
Not sure if this would be accurate? GitHub wouldn't let me add a suggestion without ruining the formatting.

Reply:
We could have:

```
kubectl edit secrets kong-mesh-license -n kong-mesh-system
```